trawler.py

```diff
@@ -166,28 +168,30 @@ def inc_counter(self, component, target_name, value, pod_name=None, labels=None)
                     metric_name = "{}.{}".format(component, target_name)
                 self.graphite.stage(metric_name, value)
 
+    def is_enabled(self, net_name):
+        """ is net in config and enabled """
+        if net_name in self.config['nets']:
+            if self.config['nets']['certs'].get('enabled', True):
```

Comment:
Is a default of `True` if it cannot find the `enabled` key correct?

Reply:
Yes that's correct